README.md

```diff
@@ -228,6 +228,47 @@ s.assert(person, {
 // person.age should be a number <= 200 (but was 250)
 ```
 
+## JSON Schema Supports
+
+Strummer can generate some simple JSON Schema from strummer definition.
+
+```js
+var schema = s({
+  foo: 'string',
+  bar: s.string({ optional: true }),
+  num: s.number({ max: 100, min: 0 })
+});
+
+console.log(schema.toJSONSchema());
+```
+
+which will shows log like this:
+
+```json
+{
+  type: 'object',
+  required: ['foo', 'num'],
+  properties: {
+    foo: {
+      type: 'string'
+    },
+    bar: {
+      type: 'string'
+    },
+    num: {
+      type: 'number',
+      maximun: 100,
+      minium: 0
+    }
+  }
+}
+```
+
+When you trying to create your own matcher which supports jsonSchema, then you needs to implements
+the `_toJSONSchema` method on your prototype, or the `toJSONSchema` option in the `createMatcher`,
+if `toJSONSchema` is not defined, when you call `matcher.toJSONSchema()` it will return nothing.
+
```

Comment:
I don't think it should return nothing, if someone creates a custom matcher without any json schema definition and it seems it's not possible for strummer to know what type the matcher belongs. For example, I didn't add the `toJSONSchema` to `s.func` matcher because in json schema doesn't have something can describe a property is a function.